test/services/pin-state-transition/hooks/prepare-activity-log.test.js

```diff
@@ -79,17 +86,37 @@ describe('Prepare Activity Log Hook for State Transition', () => {
         action: actions.ASSIGN,
         pin_id: pinId,
         changed_fields: ['status', 'assigned_department'],
-        previous_values: ['pending', undefined],
-        updated_values: ['assigned', ObjectId(DEPARTMENT_GENERAL_ID)], // eslint-disable-line new-cap,max-len
+        previous_values: ['pending', EMAIL_NOTI_NON_ASSIGNED_TEXT],
+        updated_values: ['assigned', DEPARTMENT_GENERAL_OBJECT], // eslint-disable-line new-cap,max-len
         description: `Aunt You-pin assigned pin #${PIN_PENDING_ID} ` +
                      'to department Department of Nerds',
         timestamp: Date.now(),
       };
-      expect(mockHook.data.logInfo).to.deep.equal(expectedLogInfo);
+      const actualLogInfo = mockHook.data.logInfo;
```

Comment:
Why do we have to compare field by field manually?
The old one doesn't compare nested fields?